app/dashboard/templates/funder_dashboard.html

```diff
@@ -0,0 +1,808 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include 'shared/head.html' %}
+    <link rel="stylesheet" href="{% static "v2/css/funder_dashboard.css" %}"
+  </head>
+
+  <style>
+    .text-fd-green {
+      color: #00a55e;
+    }
+
+    .text-fd-blue--light {
+      color: #519bdc;
+    }
+
+    .text-fd-blue--dark {
+      color: #3e00ff;
+    }
+
+    .text-fd-orange {
+      color: #dc9937;
+    }
+
+    .text-fd-red {
+      color: #d82a3f;
+    }
+
+    .text-fd-purple {
+      color: #8e2abe;
+    }
+
+    .text-fd-no-spacing {
+      letter-spacing: 0.100rem;
+    }
+
+    .funder-dashboard__header {
+      background-color: #3e00ff;}
```

Comment:
could we move the bracket  to the next line 
( You've already left a comment on shifting it to an external css file, so I'm good with that )